### PR TITLE
[17.0][MIG] product_harmonized_system_delivery

### PR DESCRIPTION
**Pull request by AndreuOForgeFlow**
Standard migration

Please, see this FIX PR before merging this module for optimal use: https://github.com/OCA/intrastat-extrastat/pull/268

@ForgeFlow

### CODE REVIEW

File: product_harmonized_system_delivery/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    hs_code = fields.Char(related="hs_code_id.hs_code", store=True)
+    country_of_origin = fields.Many2one(related="origin_country_id", store=True)
```
Comment (GuillemCForgeFlow) [inline product_harmonized_system_delivery/models/product_template.py:12]:
```suggestion
    # IMPORTANT: Overwrites two Odoo standard fields
    hs_code = fields.Char(related="hs_code_id.hs_code", store=True)
    country_of_origin = fields.Many2one(related="origin_country_id", store=True)
```

**Review by GuillemCForgeFlow — COMMENTED** [review]
Just this small improvement could be added I guess. The rest looks great 👍🏿

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM! 👍🏿 Thank you for adding the suggestion 😄

**Review by CarlosVForgeFlow — APPROVED** [review]
LGTM

**Review by AlexPForgeFlow — APPROVED** [review]

**Comment by GuillemCForgeFlow** [comment]
Hi @alexis-via same for this one, could we merge it? 😄

**Comment by AndreuOForgeFlow** [comment]
Hi @alexis-via, I guess this module is ready to be merged too? Also, we could add it to the Migration ToDo list.

**Pull request merged**